src/styles/card.css

```
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  width: 198px;
```

guess it should be 200 with box-sizing: border-box
```suggestion
  width: 198px;
```